ctapipe/image/tests/test_hillas.py

```diff
@@ -115,8 +115,10 @@ def test_with_toy():
 
     width = 0.03 * u.m
     length = 0.15 * u.m
+    width_uncertainty = width * 0.05
```

Comment:
How do you know the uncertainty is supposed to be 5 %?

Reply:
I have made some distributions of the uncertainties using the toy model and I have fine-tuned the expected values and tolerance for the unit test.